network/p2p/impl.go

```diff
@@ -132,7 +132,7 @@ func (c *connector) connect(ownID PeerID, config *tls.Config) (*connection, erro
 	client := transport.NewNetworkClient(grpcConn)
 	gate, err := client.Connect(cxt)
 	if err != nil {
-		log.Logger().Errorf("Failed to set up stream (peer address=%s): %v", c.address, err)
+		log.Logger().Warnf("Failed to set up stream (peer address=%s): %v", c.address, err)
```

Comment:
Also why?
And for the next 2 in this file: do we immediately have to act on these errors or not?

Reply:
I classified network connection errors as WARN because it could very well be on the peer's end, or somewhere in between. In any case you should have multiple peers (it's dangerous to 100% rely on a single peer) so connection problems with 1 peer is not something that requires immediate intention of an operator.

Maybe it'd be better to have an ERROR when you can't connect to any of your bootstrap nodes?